pkg/ccl/streamingccl/logical/logical_replication_writer_processor.go

```diff
@@ -275,19 +237,12 @@ func (lrw *logicalReplicationWriterProcessor) Start(ctx context.Context) {
 		return nil
 	})
 	lrw.workerGroup.GoCtx(func(ctx context.Context) error {
-		defer close(lrw.flushCh)
+		defer close(lrw.checkpointCh)
 		if err := lrw.consumeEvents(ctx); err != nil {
 			lrw.sendError(errors.Wrap(err, "consume events"))
 		}
 		return nil
 	})
-	lrw.workerGroup.GoCtx(func(ctx context.Context) error {
-		defer close(lrw.checkpointCh)
-		if err := lrw.flushLoop(ctx); err != nil {
-			lrw.sendError(errors.Wrap(err, "flush loop"))
-		}
-		return nil
-	})
```

Comment:
There is a block comment on this function we might want to update.

Reply:
done